helm-files.el

```diff
@@ -3470,28 +3470,28 @@ Colorize only symlinks, directories and files."
                                          (string-match ffap-url-regexp i)))
                                (not (string-match helm-ff-url-regexp i)))
                           (helm-basename i) i)
-           for type = (car (file-attributes i))
```

Comment:
Was attempting to not duplicate the check, but I was actually wondering if (file-remote-p) would be better than string-match basically everywhere?